arm-logic/2015-02-01-preview/swagger/logic.json

```diff
@@ -2023,7 +2023,10 @@
                 "Skipped",
                 "Suspended",
                 "Cancelled",
-                "Failed"
+                "Failed",
+                "Faulted",
+                "TimedOut",
+                "Aborted"
```

Comment:
I would like to let you know that this is a breaking change to the Enum. When you set the modelAsString property of the x-ms-enum extension as false then AutoRest generates an Enum in languages that support Enum (For JavaScript, the generated code does validation against the set of provided values). This is a non expandable version of the enum.

If you set the modelAsString property of the x-ms-enum extension to true then AutoRest generates a Class with Static string properties for the enum in swagger. There is no validation. (JavaScript accepts the value provided by the user and no validation is performed). This is an expandable version of the Enum where you can add/subtract the set of allowedValues. 

The modification over here is perfectly fine. This would be breaking change and we expect you to bump up the major version of the package when you publish the package.